### ISSUE TITLE
map of area affected?

### ISSUE THREAD

**Issue by derekeder** [open-city/kcbx-petcoke#20]
Might be good to map where the petcoke facility is, and the general area affected.

Maybe something like this? http://chicagococoasmell.tumblr.com/

![screen shot 2015-05-14 at 10 17 47 am](https://cloud.githubusercontent.com/assets/919583/7634742/7b771a54-fa22-11e4-9023-e6c5f656ba22.png)

**Comment by jwang207** [comment]
Good point! I was also thinking about that. Ben mentioned that part of the project once it gets going would be to have our own air sensors. For now here are some public resources for monitoring 

1, EPA with a map you can zoom in on for PM2.5 particles
http://www.epa.gov/airquality/airdata/ad_maps.html#csvfiles
1. CDC
   http://ephtracking.cdc.gov/showIndicatorPages.action

Click on "View Maps, Charts and Tables" to open the app. 

There's not a lot of info here that's recent

**Comment by justinmanley** [comment]
This is a great idea. Part of the challenge of the problem is there's very little data that we can use to define an "area affected."

One thing we _could_ do is produce a simple map showing where the petcoke piles are. @benwilhelm - do you think Olga might already have such a map?

**Comment by derekeder** [comment]
Yah start with where they are. Google Maps does the trick:

https://www.google.com/maps/place/KCBX+Terminals+Co/@41.7041069,-87.5478319,4942m/data=!3m1!1e3!4m2!3m1!1s0x880e278b44c637af:0x5119be6dec4b0d18

![screen shot 2015-05-19 at 11 09 07 pm](https://cloud.githubusercontent.com/assets/919583/7718865/49d6430e-fe7c-11e4-8a57-a12cd4e3ce15.png)
